openapi.yaml

```diff
@@ -251,6 +254,16 @@ components:
       schema:
         type: number
         minimum: 0
```

Comment:
Should it also have a `max`, as we validate the max number of cycles in `Server/Query.elm`?

Reply:
This is the `customCountryMixMakingParam` param and there's no upper limit defined by specs.